src/cookies.ts

```diff
@@ -225,6 +266,13 @@ export function createStorageFromOptions(
 
           if (cookieEncoding === "base64url") {
             encoded = BASE64_PREFIX + stringToBase64URL(value);
+          } else if (cookieEncoding === "base64url+length") {
+            encoded = [
+              BASE64_LENGTH_PREFIX,
+              value.length.toString(36),
```

Comment:
hmm what's the reason behind encoding the length as base 36 in the first place?

Reply:
also, instead of counting the total length of the reconstructed cookie, should we just count the number of chunks instead?

Reply:
> behind encoding the length as base 36 in the first place

Smaller cookie

> just count the number of chunks instead

I did consider this initially but I prefer encoding the full length as it makes future debugging a little easier because it gives us the exact length that's expected. Say there are 3 chunks 1, 2 and 3. If 2 is for some reason shorter so `len(1) + len(2) + len(3) != encoded-length` we know there's another problem.